cmse/cmse.md

```diff
@@ -0,0 +1,1725 @@
+---
+title: Arm®v8-M Security Extensions - Requirements on Development Tools
+version: Development version based on 1.1
+date-of-issue: TBD
+set-quote-highlight: true
+# LaTeX specific variables
+copyright-text: Copyright 2019-2021 Arm Limited and/or its affiliates <open-source-office@arm.com>.
+# Jekyll specific variables
+header_counter: true
+toc: true
+---
+
+<!--
+SPDX-FileCopyrightText: Copyright 2021 Arm Limited and/or its affiliates <open-source-office@arm.com>
```

Comment:
The specs in this file where copyrighted in 2019, so we need this to change too:
```suggestion
SPDX-FileCopyrightText: Copyright 2019, 2021 Arm Limited and/or its affiliates <open-source-office@arm.com>
```